pkg/hubble/observer/local_observer.go

```diff
@@ -415,15 +415,14 @@ func (r *flowsReader) Next(ctx context.Context) (*observerpb.GetFlowsResponse, e
 // newRingReader creates a new RingReader that starts at the correct ring
 // offset to match the flow request.
 func newRingReader(ring *container.Ring, req *observerpb.GetFlowsRequest, whitelist, blacklist filters.FilterFuncs) (*container.RingReader, error) {
-	if req.Follow && req.Number == 0 { // no need to rewind
+	if req.Follow && req.Number == 0 && req.Since == nil { // no need to rewind
```

Comment:
`until 24h ago` is a valid rewind scenario, no?

Reply:
Yes, good catch. I don't think the CLI supports `until`+`follow` right now, but no reason to not support it.

Reply:
I don't think `until`+`follow` is a very common use-case. I think I misread the code, I was just trying to make sure that `until 24ago no follow` would rewind properly

Reply:
Yes, so turns out we want to rewind (or not rewind) regardless of the value of  `req.Until`. `until` doesn't tell us how much we want to rewind (reading backwards), it only tells us when to stop when reading forward. So I guess the condition was correct in either case.

I believe the server-side implementation as of this PR will actually work with `follow`+`until`, but again this likely needs CLI changes first (as the CLI currently clears until if follow-mode is set) for users to actually be able to use it.